packages/tools/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this tool will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [9.18.0] - 2022-08-24
+
+### Deprecated
+
+-   We've deprecated the `className` preview property. From now on, you have to use `class` property instead.
```

Comment:
A version header is placed automatically, no need to place it manually. There is `[Unreleased]` section, all changelogs should go under that section, as if it is a version. The release script have to take care of the rest.